packages/vscode-extension/src/project/deviceSession.ts

```diff
@@ -279,15 +278,11 @@ export class DeviceSession
       this.emitStateChange();
     });
     devtools.onEvent("fastRefreshStarted", () => {
-      this.status = "refreshing";
+      this.isRefreshing = true;
       this.emitStateChange();
     });
     devtools.onEvent("fastRefreshComplete", () => {
-      const ignoredEvents = ["starting", "bundlingError"];
-      if (ignoredEvents.includes(this.status)) {
-        return;
-      }
-      this.status = "running";
+      this.isRefreshing = false;
```

Comment:
this is bad that we only ever set it to `false` here. If we miss the event (i.e. the app would crash when refreshing, or someone reloads the app while refreshing), it will never reset. We should at least set it to false in `resetStartingState` method and perhaps in some other places too?

Reply:
`resetStartingState` seems to be enough, since it's called wherever the app is reloaded.